server/src/server.js

```diff
@@ -6,7 +6,7 @@ import cors from "cors"
 dotenv.config()
 const app = express()
 
-app.use(express.json())
+app.use(express.json({ limit: 4000000 }))
```

Comment:
😆

Reply:
I saw this too — is this because we're returning the image as a string or a buffer or something?

Reply:
`req.body` has a size limitation of 100kb. Since he's sending a image binary file to his backend endpoint, he has to increase the limit.

Reply:
it;s over 9000!!!